github/codeowners.py

```diff
@@ -84,6 +88,107 @@ def _first(iterable):
         return None
 
 
+@dataclasses.dataclass(frozen=True)
+class CodeOwnerGithubUser:
+    user: str
+    source: str
+
+
+@dataclasses.dataclass(frozen=True)
+class CodeOwnerPersonalName:
+    firstName: str
+    lastName: str
+    source: str
+
+
+@dataclasses.dataclass(frozen=True)
+class CodeOwnerEmail:
+    email: str
+    source: str
+
+
+@dataclasses.dataclass
+class CodeOwner:
```

Comment:
that's not exactly how I imagined this :-/

also, I think this code actually belongs into the delivery-service repository

I would like to have a more abstract object representing a UserIdentity (maybe call it like this?)

A UserIdentity consists of a non-empty collection of (typed) UserInfo objects:
- email-address
- github-user
- personal-name
- ... (for now, there are no more, but there might be. e.g. "d-user")